orc8r/cloud/go/services/state/indexer/reindex/reindexer_singleton_test.go

```diff
@@ -163,5 +184,39 @@ func initSingletonReindexTest(t *testing.T) reindex.Reindexer {
 		reportGatewayStatus(t, ctxByNetwork[nid], gwStatus)
 	}
 
-	return reindexer
+	ctx, cancel := context.WithCancel(context.Background())
+	go reindexer.Run(ctx)
+
+	return cancel, reindexer
+}
+
+func addNetworks(t *testing.T, reindexer reindex.Reindexer) {
+	ctx, _ := context.WithCancel(context.Background())
+	go reindexer.Run(ctx)
```

Comment:
Hoist this outside this func -- running this indexer is unrelated to reporting more state

Reply:
It is related since we run `Run` before reporting more state

Reply:
Ahhh I see what you mean